src/generateRandomDigits.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const { random } = require('lodash');
+
+function generateDigits() {
+  let digits = '';
+
+  while (digits.length < 4) {
+    const randomInt = random(0, 9).toString();
+
+    if (!digits.includes(randomInt)) {
```

Comment:
Consider using `Set` to check if the random integer is already in the `digits` string instead of `includes` method. 🤔